app/src/main/res/layout/activity_settings.xml

```diff
@@ -11,6 +11,7 @@
         android:layout_height="wrap_content"
         android:layout_marginStart="8dp"
         android:layout_marginEnd="8dp"
+        android:layout_marginTop="8dp"
```

Comment:
It's not a good habit to hardcode dimens value in a big project. Use diemns file to store the value and use them in the xml file.

Reply:
No resources are in dimens file. Everywhere it is hardcoded. It is not something to be done for one file. Please open a new issue about work for the whole project's dimensions. Let me know if you want to work on that otherwise, I'll proceed. Thanks.

Reply:
I think we should maybe define our dimens to be used through the app 🤔 
I will bring it to the design team I think, then we can refactor the whole app @atm1504, what do you think?

Reply:
@isabelcosta I agree with you. Dimensions should be uniform throughout the app. The design team can surely help.